resources/views/layouts/partials/header.blade.php

```diff
@@ -1,29 +1,57 @@
-<header class="flex flex-wrap items-center mb-5 border-b shadow {{ Route::currentRouteName() == 'checkout' ? 'justify-between' : '' }}">
-    <div class="w-1/6 sm:w-3/12">
-        <div class="text-xl sm:text-3xl ml-3">
-            <a href="/" aria-label="@lang('Go to home')">
-                <span class="hidden sm:inline">
-                    <img src="https://raw.githubusercontent.com/rapidez/art/master/logo.svg" height="48" width="152" alt="">
-                </span>
-                <span class="inline sm:hidden">🚀</span>
-            </a>
+<header class="relative mb-5 border-b shadow">
+    <div class="{{ Route::currentRouteName() == 'checkout' ? 'justify-center' : '' }} container mx-auto flex flex-wrap items-center max-sm:px-3">
+        <input
+            id="navigation"
+            class="peer hidden"
+            type="checkbox"
+        />
+        <div class="py-2">
+            <div class="mr-3 flex items-center text-xl sm:text-3xl">
+                <a
+                    href="/"
+                    aria-label="@lang('Go to home')"
+                >
```

Comment:
Maybe recheck the indenting everywhere?

```suggestion
                <a href="/" aria-label="@lang('Go to home')">
```